testing/tests/plugin

```diff
@@ -3,21 +3,23 @@
 # @TEST-EXEC: bash %INPUT
 # @TEST-EXEC: zkg install rot13
 
-# @TEST-EXEC: test -f plugins/packages/rot13/__bro_plugin__
+# @TEST-EXEC: test -f plugins/packages/rot13/__bro_plugin__ || test -f plugins/packages/rot13/__zeek_plugin__
 # @TEST-EXEC: btest-diff scripts/packages/rot13/__load__.zeek
 
 # Unloading the package should also disable the plugin, which we
 # detect via the renamed __bro_plugin__ magic file.
 # @TEST-EXEC: zkg unload rot13
 
 # @TEST-EXEC: test ! -f plugins/packages/rot13/__bro_plugin__
-# @TEST-EXEC: test -f plugins/packages/rot13/__bro_plugin__.disabled
+# @TEST-EXEC: test ! -f plugins/packages/rot13/__zeek_plugin__
```

Comment:
Nit: I'd go with the following for symmetry with all the other fallbacks (after all this is just a single check against multiple versions).

```zeek
# @TEST-EXEC: test ! -f plugins/packages/rot13/__bro_plugin__ && test ! -f plugins/packages/rot13/__zeek_plugin__
```

Reply:
Thanks!